src/main/java/cloud/eppo/Utils.java

```diff
@@ -16,6 +14,7 @@ public final class Utils {
   private static final SimpleDateFormat UTC_ISO_DATE_FORMAT = buildUtcIsoDateFormat();
   private static final Logger log = LoggerFactory.getLogger(Utils.class);
   private static final MessageDigest md = buildMd5MessageDigest();
+  public static Base64Codec base64Codec;
```

Comment:
How about making this private with a setter

Reply:
done